src/domainPaths/domainPath.ts

```diff
@@ -179,10 +179,11 @@ export class DomainPath {
    *
    * @returns the generated {@link IAuthlessResponse}
    */
-  public async convertPageToResponse (page: PuppeteerPage, mainResponse: PuppeteerResponse): Promise<IAuthlessResponse> {
+  public async convertPageToResponse (page: PuppeteerPage, mainResponse: PuppeteerResponse, bot: Bot): Promise<IAuthlessResponse> {
```

Comment:
I think this method should not be a `DomainPath` method. What this method really does is, is construct a `AuthlessResponse` instance. So feel like it would be more appropriate to make it part of `AuthlessResponse` directly. Like so:

```typescript
class AuthlessResponse {

  ...

  public static async fromPage (page: PuppeteerPage, mainResponse: PuppeteerResponse, bot: Bot): Promise<AuthlessResponse> {
     ...
  }
}
```